versioned_docs/version-8.2/self-managed/platform-deployment/helm-kubernetes/upgrade.md

```diff
@@ -5,14 +5,12 @@ sidebar_label: "Upgrade"
 description: "To upgrade to a more recent version of the Camunda Helm charts, there are certain things you need to keep in mind."
 ---
 
-To upgrade to a more recent version of the Camunda Helm charts, there are certain things you need to keep in mind.
-
-:::caution
-
-Ensure to review the [instructions for specific version](#version-update-instructions) before starting the actual upgrade.
-
+:::note
+When upgrading to a new version of the Camunda 8 Helm charts, we recommend updating to the **latest patch** release of the next **minor** version of the chart.
 :::
 
+To upgrade to a more recent version of the Camunda Helm charts, there are certain things you need to keep in mind. Review the [instructions for a specific version](#version-update-instructions) before starting the actual upgrade.
```

Comment:
```suggestion
Before upgrading to a more recent version of the Camunda Helm charts, you should review the [instructions for a specific version](#version-update-instructions).
```
Just a rewording suggestion for conciseness - wasn't sure whetehr to use "should" or "must" either?